source/Clipboard.js

```diff
@@ -273,6 +273,9 @@ var onDrop = function ( event ) {
     var l = types.length;
```

Comment:
There still isn't a 100% reliable way to do this across browsers, as far as I can tell.